### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by Awzaw**

### CODE REVIEW

File: doc/mysql.sql [hunk 1]
```diff
@@ -48,9 +48,9 @@ CREATE TABLE builds (
     projectId INT REFERENCES projects(projectId),
     class TINYINT, -- Dev = 1, Beta = 2, Release = 3
     branch VARCHAR(255) DEFAULT 'master',
-    cause VARCHAR(16383),
+    cause VARCHAR(8192),
     internal INT, -- internal (project,class) build number, as opposed to global build number
-    status VARCHAR(32767) DEFAULT '[]',
+    status VARCHAR(8192) DEFAULT '[]',
```
Comment (SOF3) [inline doc/mysql.sql]: Actually should I put the status in a resource JSON file? It seems really limited here.

File: src/poggit/module/releases/index/PluginsByRepoReleaseListPage.php [hunk 2]
```diff
@@ -42,22 +42,22 @@ public function __construct(string $query, array $filters) {
...
                 $args[] = $pieces[0];
                 $args[] = $pieces[1];
                 $semiTitlesIn[] = "$pieces[0]/$pieces[1]";
             }
         }
-        $where = implode(" OR ", $wheres);
+        $where = "(" . implode(" OR ", $wheres) . ")";
         $plugins = Poggit::queryAndFetch("SELECT 
-            r.releaseId, r.name, r.version, rp.owner AS author, r.shortDesc,
+            r.releaseId AS releaseId, r.name AS name, r.version AS version, rp.owner AS author, r.shortDesc AS shortDesc,
```
Comment (SOF3) [inline src/poggit/module/releases/index/PluginsByRepoReleaseListPage.php]: These work too.

Comment (Awzaw) [inline src/poggit/module/releases/index/PluginsByRepoReleaseListPage.php]: Indeed, that was leftover from my tests why the query was failing. Can't do any harm though!

File: doc/mysql.sql (hunk 1 above)
Comment (Awzaw) [inline doc/mysql.sql]: Depends what you're going to use it for I suppose, but it seems to belong here I think

Comment (SOF3) [inline doc/mysql.sql]: Or a separate table for statuses, but I want to make the database more complex.

**Comment by SOF3** [comment]
Notable issues are fixed locally. I will push them later.

**Pull request closed without merging**